content/project-news/hermes-belusca-hired-full-time.md

```diff
@@ -0,0 +1,27 @@
+---
+title:       "Hermès BÉLUSCA - MAÏTO hired full-time to work on the ReactOS GUI Setup"
+author:      "GeoB99"
+date:        2023-09-28
+---
+
+Our team proudly announces that ReactOS Deutschland e.V. has hired Hermès BÉLUSCA - MAÏTO to work full-time on the ReactOS GUI Setup for the next 3 months.
```

Comment:
```suggestion
Our team proudly announces that ReactOS Deutschland e.V. has hired Hermès Bélusca-Maïto to work full-time on the ReactOS GUI Setup for the next 4 months.
```
Besides the casing, I've adjusted the duration here for "4" months. I based myself on what was agreed (see ros-priv emails), namely 5 months starting on beginning september, and here in the announcement I removed 1 month because that month will expire by the time this announcement is published :)

Please tell me whether you want to re-adjust the sentence so that it instead says, that I've started on September 2, and that the duration of the contract is 5 months.
In that case, that sentence could read:
```suggestion
Our team proudly announces that ReactOS Deutschland e.V. has hired Hermès Bélusca-Maïto to work full-time on the ReactOS GUI Setup for the next 5 months, starting September 2, 2023.
```

Reply:
I'd suggest to include the start date and make it 5 months, because it better reflects the actual status.